scss/components/_form-styles.scss

```diff
@@ -230,3 +230,75 @@
     border: 1px solid $error;
   }
 }
+
+
+// Inline range inputs
+//
+// Half width inputs so min and max filters display side by side
+//
+
+.range {
+  margin-bottom: u(1rem);
+  position: relative;
+
+  .js-filter {
```

Comment:
Can we use a different class here, so we're not using `.js-*` classes for styles?